docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -74,7 +74,7 @@
             <td><h5>parallelism.default</h5></td>
             <td style="word-wrap: break-word;">1</td>
             <td>Integer</td>
-            <td>Default parallelism for jobs.</td>
+            <td>Default parallelism for jobs. There are two special case. The first case is when creating a StreamExecutioneEnvironment, if the value of parallelism.default is not explicitly specified, the number of processors available to the Java virtual machine will be used as the default parallelism. The second case is when creating mini cluster, the number of slots will be used as the default parallelism</td>
```

Comment:
nits:
 "There are two special case." -> "There are two cases."
"The first case is when creating a StreamExecutioneEnvironment, if the value of parallelism.default is not explicitly specified, the number of processors available to the Java virtual machine will be used as the default parallelism." ->
In the first case, when creating a StreamExecutioneEnvironment without specifying  parallelism.default ,the number of processors available to the Java virtual machine will be used."

" The second case is when creating mini cluster, the number of slots will be used as the default parallelism"
-> I am not sure what you mean by mini cluster. The slots are on a task manager - how does this relate?

Reply:
It's the total number of slots in mini cluster.